tests/ui/feature-gates/feature-gate-super-let.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    super let a = 1;
+    //~^ ERROR `super let` is experimental
+}
```

Comment:
I don't want to run this back through CI, but in follow-up work, let's please remember to add a test that confirms we get this error when the code is `cfg`ed out.  That's correctly done here (it's apparent in the code and I tested locally), but we should always want a test for this.